ubcpi/static/js/spec/ubcpi_edit_spec.js

```diff
@@ -277,14 +283,29 @@ describe('UBCPI_Edit module', function () {
             controller.deleteSeed(1);
             expect(controller.data.seeds.length).toBe(num_seeds - 1);
             expect(controller.data.seeds).toEqual([
+                {answer:2, rationale:'rationale3'},
                 {answer:0, rationale:'rationale1'},
-                {answer:2, rationale:'rationale3'}
+                {answer:2, rationale:'rationale3'},
+                {answer:1, rationale:'rationale2'},
+                {answer:0, rationale:'rationale1'}
             ]);
         });
 
+        it('should delete seeds containing an option when delete_option is called', function(){
+            var num_seeds = mockConfig.data.seeds.length;
+            controller.delete_option(2);
```

Comment:
Could you change this to delete option 1 or add another test to test the seed answer re-indexing?